cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -989,3 +989,19 @@ def test_ssa_multiple_vms(ssa_multiple_vms, soft_assert, appliance, compare_linu
             compare_linux_vm_data(ssa_vm)
         else:
             compare_windows_vm_data(ssa_vm)
+
+
+@pytest.mark.manual("manualonly")
+def test_ssa_vm_ec2_agent_tracker():
```

Comment:
Missing: ```@pytest.mark.meta(coverage=[1557452])```

Reply:
@ganeshhubale I have just moved test from one file to another, have plan to automate it.